RLTest/__main__.py

```diff
@@ -171,7 +171,13 @@ def do_normal_conn(self, line):
     help='stop before each test allow gdb attachment')
 
 parser.add_argument(
-    '-t', '--test', help='Specify test to run, in the form of "file:test"')
+    '-t', '--test', action='append', help='test to run, in the form of "file:test"')
+
+parser.add_argument(
+    '-f', '--tests-file', action='append', help='file containing test to run, in the form of "file:test"')
```

Comment:
Can you see if this parser supports ```metavar=FILE```

Reply:
This is just for aesthetics of help message. Tested in with RedisGraph, looks ok.